zeebe/stream-platform/src/test/java/io/camunda/zeebe/stream/impl/RecordBatchTest.java

```diff
@@ -159,7 +159,7 @@ void shouldOnlyAddUntilMaxBatchSizeIsReached() {
     assertThat(either.isLeft()).isTrue();
     assertThat(either.getLeft())
         .hasMessageContaining("Can't append entry")
-        .hasMessageContaining("[ currentBatchEntryCount: 1, currentBatchSize: 326]");
+        .hasMessageContaining("[ currentBatchEntryCount: 1, currentBatchSize: 334]");
```

Comment:
@Zelldon I updated the batch sized here and below to pass the failing tests. I assume this is fine, right?

Reply:
Might make sense to completely remove this from the assert, and only check for the count and parts of the message 🤔

Reply:
I think if I remove the sizes there the whole test would be empty, and I'm not very confident with this part to create more meaningful ones. I'm happy to pair on it together though if you'd like that.

Reply:
This one has a count indeed, but the one below `shouldOnlyReturnTrueUntilMaxBatchSizeIsReached` has only a size predicate passed.
I guess it's testing that if we appened one record that fits the maximum size of a batch, we can't append another one. It feels a bit redundant to me though.